interfaces/kmod/kmod.go

```diff
@@ -18,3 +18,61 @@
  */
 
 package kmod
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/snapcore/snapd/dirs"
+	"github.com/snapcore/snapd/interfaces"
+	"github.com/snapcore/snapd/osutil"
+	"github.com/snapcore/snapd/snap"
+)
+
+func moduleFilePath(snapName string) string {
+	confFileName := fmt.Sprintf("%s.conf", snap.SecurityTag(snapName))
```

Comment:
@jdstrand Yes, I know we discussed creating a file per-interface. After investigating it more in the context of existing security backend interface in the code I found it akward though, as the model we have (with backends providing security snippets for given snap) simply didn't match. I had a conversation with zyga and he agreed a .conf file per snap is fine. Given that - according to systemd's manual - /etc/modules.d acts as a place where vendors and arbitrary packages can drop their .conf files,  having duplicated modules in there is not an issue.